### ISSUE TITLE
BUG CurlHttpAdapter referenced in GeocoderAdapter although not required via composer.json

### ISSUE THREAD

**Issue by muskie9** [dynamic/silverstripe-geocoder#56]
My guess is `egeloen/http-adapter` was a requirement of a package this module is requiring but no longer is defined. Short term we can add this in as a requirement of this module but we should understand why it may have been dropped from other packages as a requirement and explore alternatives.

**Comment by muskie9** [comment]
This is related to the recent updates for PHP 8 compatibility. There are a few packages that may be missing due to changes in currently required packages.

**Comment by obj63mc** [comment]
Also noting that the geocoder provider is also missing when installing the latest v1.3 - 

Example error - 

```
ERROR [Emergency]: Uncaught Error: Class "Geocoder\Provider\GoogleMaps" not found
IN GET dev/tasks/RetailerTask
Line 32 in /Users/joemadden/Sites/talon-cigars/vendor/dynamic/silverstripe-geocoder/src/GoogleGeocoder.php
```

**Comment by jsirish** [comment]
closed via #57
